neural_lam/utils.py

```diff
@@ -72,14 +78,26 @@ def loads_file(fn):
         device=device,
     )  # (d_f,)
 
+    raw_coords = np.load(os.path.join(static_dir_path, "nwp_xy.npy"))
+    interior_coords = raw_coords.reshape(2, -1)[:, interior_mask.numpy()]
+    grid_limits = [
+        interior_coords[0].min(),
+        interior_coords[0].max(),
+        interior_coords[1].min(),
+        interior_coords[1].max(),
+    ]
+
     return {
-        "border_mask": border_mask,
+        "boundary_mask": boundary_mask,
+        "interior_mask": interior_mask,
         "grid_static_features": grid_static_features,
+        "boundary_static_features": boundary_static_features,
         "step_diff_mean": step_diff_mean,
         "step_diff_std": step_diff_std,
         "data_mean": data_mean,
         "data_std": data_std,
         "param_weights": param_weights,
+        "grid_limits": grid_limits,
```

Comment:
The masks and grid limits here are only temporary ways to handle this. 

* Masks should not be needed once we have dataloading capable of loading inputs from separate interior and boundary sources. 
* The grid limits should come from the coordinate information in a datastore.